tests/settings/mtz_event_config.yaml

```diff
@@ -50,13 +50,13 @@ PHONE_DATA_STREAMS:
   
   # AVAILABLE:
   aware_mysql: 
-    DATABASE_GROUP: MY_GROUP
+    DATABASE_GROUP: RHYTHMS
```

Comment:
Change all the `DATABASE_GROUP` fields back to `MY_GROUP`, please.

Reply:
Changed!